src/main/java/string/Exercise06.java

```diff
@@ -0,0 +1,25 @@
+package string;
+
+public class Exercise06 {
+    public boolean checkBarcode(final String code) {
+        int sumEven = 0;
+        int sumOdd =0;
+        if (code == null || code.length() != 13) {
+            return false;
+        }
```

Comment:
Câu lệnh if này có thể được thực hiện mà không cần 2 biến trên, do đó cần đổi lại


```suggestion
        
        if (code == null || code.length() != 13) {
            return false;
        }
        
        int sumEven = 0;
        int sumOdd =0;
        
```

Chú ý dòng trống